tests/test_14_read_only_list_file.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from idpyoidc.storage.listfile import ReadOnlyListFile
+
+BASEDIR = os.path.abspath(os.path.dirname(__file__))
+
+
+def full_path(local_file):
+    return os.path.join(BASEDIR, local_file)
+
+FILE_NAME = full_path("read_only")
+def test_read_only_list_file():
+    if os.path.exists(FILE_NAME):
+        os.unlink(FILE_NAME)
+        os.unlink(f"{FILE_NAME}.lock")
+
+    _read_only = ReadOnlyListFile(FILE_NAME)
+    assert len(_read_only) == 0
+
+    with open(FILE_NAME, "w") as fp:
+        for item in ["one", "two", "three"]:
+            fp.write(item)
+            fp.write("\n")
```

Comment:
```suggestion
        fb.writelines(["one", "two", "three"])
```

Reply:
The '\n' at the end of the line is significant and writelines doesn't add those so I adopted a variant of what you proposed.